npm-cli.adoc

```diff
@@ -0,0 +1,133 @@
+= NPM CLI -  Commands and Flags
+
+This cheat sheet covers the most common and helpful commands and flags for the npm CLI.
+
+== Basic Usage
+
+`-v, --version`:: Prints the current version of npm you are using
+`<command> -h, --help`:: quick help on the specified command
+`-l`:: display full usage info
+
+`npx [command]`:: Run an arbitrary command from an npm module (either one installed locally, or fetched remotely)
```

Comment:
This still works in npm 7 but I believe the npm team are trying to steer towards `npm exec` which is npm 7 only.